Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -473,6 +473,8 @@ The list of available options is reported in the :ref:`table <tab:cmakevar>` bel
    +------------------------------+-------------------------------------------------+-------------------------+-----------------------+
    | AMReX_PROFPARSER             |  Build with profile parser support              | NO                      | YES, NO               |
    +------------------------------+-------------------------------------------------+-------------------------+-----------------------+
+   | AMReX_ROCTX                  |  Build with roctx markup profiling support      | NO                      | YES, NO               |
```

Comment:
```suggestion
   | AMReX_ROCTX                  |  Build with roctx markup profiling support      | ON (if HIP)                    | YES, NO               |
```

Reply:
My view is we should simply remove this option in the future once it's fixed on spock.